commands.c

```diff
@@ -1068,12 +1068,14 @@ int mutt_save_message(struct Mailbox *m, struct EmailList *el, bool delete,
 #endif
 
   struct Mailbox *m_save = mx_path_resolve(buf);
-  struct Context *ctx_save = mx_mbox_open(m_save, MUTT_APPEND);
+  struct Context *ctx_save = mx_mbox_open(m_save, 0);
```

Comment:
[0] Since we've gone through the trouble of `typedef`ing `OpenMailboxFlags`, shouldn't this be `MUTT_OPEN_NO_FLAGS`?

Reply:
Yeah :-)
These "new" commits have actually been sitting around for a while
(they originally predated the `OpenMailboxFlags` changes).